roles/st2/tasks/main.yml

```diff
@@ -1,22 +1,10 @@
 ---
 
-- name: Install latest st2 package
-  become: yes
-  apt:
-    name: st2
-    state: latest
-  when: st2_version == "stable"
-  notify:
-    - restart st2
+- include: debian.yml
+  when: ansible_os_family == 'Debian'
```

Comment:
We can do 1 include without `when` condition.

See @humblearner arppoach https://github.com/StackStorm/ansible-st2/blob/3785d849135599de1f667e692576d3501af67fb5/roles/mongodb/tasks/main.yml#L3 with 2 files `mongodb_apt.yml` and `mongodb_yum.yaml` which looks nicer and cleaner.

It would be better if we follow one standard in all roles for consistency.